Tests/momentoTests/testUtils.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+@testable import momento
+
+struct TestSetup {
+    var cacheName: String
+    var cacheClient: CacheClientProtocol
+    var topicClient: TopicClientProtocol
+}
+
+func setUpIntegrationTests() async -> TestSetup {
+    do {
+        let creds = try CredentialProvider.fromEnvironmentVariable(envVariableName: "TEST_AUTH_TOKEN")
```

Comment:
```suggestion
        let creds = try CredentialProvider.fromEnvironmentVariable(envVariableName: "TEST_API_KEY")
```

We are using the terminology API key now 👍